src/components/UserInfo.js

```diff
@@ -2,6 +2,7 @@ export default class UserInfo {
   constructor({ nameSelector, jobSelector }) {
     this._nameElement = document.querySelector(nameSelector);
     this._jobElement = document.querySelector(jobSelector);
+    this._userAvatarElement = document.querySelector(".profile__image");
```

Comment:
`".profile__image"` should be passed as `avatarSelector` into `UserInfo` not to hardcode the string

Reply:
so instead of _userAvatarElement i should write avatarSelector?

Reply:
> so instead of _userAvatarElement i should write avatarSelector?

No

I mean this

```js

export default class UserInfo {
  constructor({ nameSelector, jobSelector, avatarSelector }) {
    this._nameElement = document.querySelector(nameSelector);
    this._jobElement = document.querySelector(jobSelector);
    this._userAvatarElement = document.querySelector(avatarSelector);
  }
```